styles.css

```diff
@@ -482,3 +497,598 @@ textarea {
   width: 100%;
   margin: 124px 0 24px 0;
 }
+
+@media (min-width: 992px) and (max-width: 1100px) {
```

Comment:
- [ ] [OPTIONAL] Good job addressing the previous reviewer suggestions but when mentioned that you should use a value between `992px` and `1100px` it was to apply one value, e.g., `min-width: 1080px`, and this "breakpoint" will be applied to all larger devices and it's not necessary to add the `max-width`. You could include all the properties for larger devices just here and remove the `max-width` :memo: :pushpin:

Reply:
Thanks, now are clear.